docs/Reference.md

```diff
@@ -89,7 +89,7 @@ timestamp_ms - DISCORD_EPOCH << 22
 
 There are some cases in which our API and Gateway may return IDs in an unexpected format. Internally, Discord stores IDs as integer snowflakes. When we serialize IDs to JSON, we transform `bigints` into strings. Given that all Discord IDs are snowflakes, you should always expect a string.
 
-However, there are cases in which passing something to our API will instead return you IDs serialized as an integer; this is the case when you send our API or Gateway a value in an `id` field that is not `bigint` size. For example, when requesting `GUILD_MEMBERS_CHUNK` from our gateway:
```

Comment:
This change is actually not quite correct. It's not returning your IDs, it's returning IDs to you, so the documentation as-is is correct